### PR TITLE
fix(web): various UI fixes

### PR DESCRIPTION
**Pull request by denis-kralj-novu**
# video showing the current state

https://www.loom.com/share/ec4d1505251745e8949cbcfae0ca5cf6

### CODE REVIEW

File: apps/web/src/ee/clerk/components/UserProfileButton.tsx
Comment (antonjoel82) [inline apps/web/src/ee/clerk/components/UserProfileButton.tsx]:
```suggestion
          Billing plans
```

File: apps/web/src/ee/clerk/providers/ClerkProvider.tsx
Comment (antonjoel82) [inline apps/web/src/ee/clerk/providers/ClerkProvider.tsx]: ☑ **chore:** Move outside the component. Can even move to a `ClerkProvider.styles.ts` (or `ClerkProvider.const.ts`) file, but we don't want these redefined on every render.‏

File: apps/web/src/ee/clerk/providers/ClerkProvider.tsx
Comment (antonjoel82) [inline apps/web/src/ee/clerk/providers/ClerkProvider.tsx:1]: ☑ **chore:** Please try removing all `!important` and only keep ones that are ultimately necessary‏

File: apps/web/src/ee/clerk/providers/ClerkProvider.tsx
Comment (antonjoel82) [inline apps/web/src/ee/clerk/providers/ClerkProvider.tsx]: ☑ **chore:** Please use `zIndex` token‏

**Review by SokratisVidros — APPROVED** [review]

File: apps/web/src/ee/clerk/providers/ClerkProvider.tsx
Comment (antonjoel82) [inline apps/web/src/ee/clerk/providers/ClerkProvider.tsx]: ☑ **chore:** Please define a semantic token in `web/panda.config.ts` and use here‏

**Review by antonjoel82 — APPROVED** [review]
Nice work on this! Approving to unblock, but please consider doing the chores before merging as time permits

File: apps/web/src/ee/clerk/components/UserProfileButton.tsx
Comment (denis-kralj-novu) [reply apps/web/src/ee/clerk/components/UserProfileButton.tsx]: done

File: apps/web/src/ee/clerk/providers/ClerkProvider.tsx
Comment (denis-kralj-novu) [reply apps/web/src/ee/clerk/providers/ClerkProvider.tsx]: done

File: apps/web/src/ee/clerk/providers/ClerkProvider.tsx
Comment (denis-kralj-novu) [reply apps/web/src/ee/clerk/providers/ClerkProvider.tsx:1]: done, none were !important (*nudge nudge, wink wink)

File: apps/web/src/ee/clerk/providers/ClerkProvider.tsx
Comment (denis-kralj-novu) [reply apps/web/src/ee/clerk/providers/ClerkProvider.tsx]: opted to skip for now

File: apps/web/src/ee/clerk/providers/ClerkProvider.tsx
Comment (denis-kralj-novu) [reply apps/web/src/ee/clerk/providers/ClerkProvider.tsx]: opted to skip for now

**Pull request merged**
